composer.json

```diff
@@ -52,6 +52,9 @@
 		"check-cs": [
 			"\"vendor/bin/phpcs\""
 		],
+		"check-cs-errors": [
+			"\"vendor/bin/phpcs\" --error-severity=1 --warning-severity=6"
```

Comment:
> I just needed to have a quick command in my terminal to only see the errors while checking

That's what the `-n` command line flag is for ;-)

> I added --warning-severity=6 because default warnings seem to be triggered at 5

Correct, same goes for errors. Setting severity to `0` disables the display of something (can be done for a specific errorcode, sniff etc).